sites/svelte.dev/src/routes/apps/index.svelte

```diff
@@ -55,6 +79,8 @@
 			</div>
 		</header>
 
+		<input placeholder="Search apps..." bind:value={searchValue} />
```

Comment:
This needs a label for a11y -- either an actual `<label>` or an `aria-label` attribute.